renet_visualizer/src/lib.rs

```diff
@@ -366,9 +366,11 @@ fn show_graph(
 
         let body = Shape::Rect(RectShape {
             rect,
-            rounding: Rounding::none(),
+            rounding: Rounding::ZERO,
             fill: Rgba::TRANSPARENT.into(),
             stroke: style.rectangle_stroke,
+            uv: Rect::ZERO,
+            fill_texture_id: egui::TextureId::Managed(0),
```

Comment:
As far as I could tell this was the only "code change" that egui 0.22 -> 0.23 required